k8s/modules-cid-ckecker.tf

```diff
@@ -0,0 +1,29 @@
+module "marketdeals_calibration" {
+  count           = local.is_dev_envs
+  source          = "../modules/cid_checker"
+  bucket_name     = "calibration"
+  force_destroy   = true
+  tags            = module.generator.common_tags
+  oidc            = local.oidc_URL
+  name_prefix     = module.generator.prefix
+}
+
+module "marketdeals_mainnet" {
+  count           = local.is_mainnet_envs
+  source          = "../modules/cid_checker"
+  bucket_name     = "mainnet"
+  force_destroy   = true
+  tags            = module.generator.common_tags
+  oidc            = local.oidc_URL
+  name_prefix     = module.generator.prefix
+}
+
+module "marketdeals_wallaby" {
+  count           = local.is_dev_envs
+  source          = "../modules/cid_checker"
+  bucket_name     = "wallaby"
+  force_destroy   = true
+  tags            = module.generator.common_tags
+  oidc            = local.oidc_URL
+  name_prefix     = module.generator.prefix
```

Comment:
I would suggest executing the `name_prefix` directly in the module.

Reply:
Removed variable name_prefix  as included general module into cid-checker.